pinot-core/src/test/java/com/linkedin/pinot/query/selection/SelectionOnlyQueriesForMultiValueColumnTest.java

```diff
@@ -208,27 +206,12 @@ public void testInnerSegmentPlanMakerForSelectionNoFilter() throws Exception {
     instanceResponseMap.put(new ServerInstance("localhost:8888"), resultBlock.getDataTable());
     instanceResponseMap.put(new ServerInstance("localhost:9999"), resultBlock.getDataTable());
     final Collection<Serializable[]> reducedResults =
-        SelectionOperatorUtils.reduce(instanceResponseMap, brokerRequest.getSelections().getSize());
+        SelectionOperatorUtils.reduceWithoutOrdering(instanceResponseMap, brokerRequest.getSelections().getSize());
```

Comment:
Could you port the test to use BrokerResponseNative, instead of removing the check?